pkgs/development/libraries/apr/default.nix

```diff
@@ -44,8 +48,24 @@ stdenv.mkDerivation rec {
 
   configureFlags = lib.optional (stdenv.hostPlatform != stdenv.buildPlatform) [
     "ac_cv_file__dev_zero=yes"
-    "ac_cv_func_setpgrp_void=0"
-    "apr_cv_tcp_nodelay_with_cork=1"
+    "ac_cv_func_setpgrp_void=yes"
```

Comment:
We really should add a comment why we set all those options.

Reply:
Added an extra comment about the configure test answers.